app/db/crud.py

```diff
@@ -0,0 +1,81 @@
+from uuid import UUID
+from sqlalchemy import select
+from sqlalchemy.orm import Session
+from fastapi import HTTPException
+from app.db.models.builder import Keycap, Switch, Lubricant, Kits, Builds
+import typing
+
+
+def get_keycaps(db: Session) -> list[typing.Any]:  # type: ignore
```

Comment:
Type annotations for functions and methods should not be `type: ignored`. Should not be using `Any`